### ISSUE TITLE
Integrated Console prompt problem when posh-git is loaded

### ISSUE THREAD

**Issue by csandfeld** [PowerShell/vscode-powershell#760]
### System Details

- Operating system name and version:
Windows 10 Enterprise Version 1607 (OS Build 14393.1066)

```
PS C:\> code -v

1.12.2
19222cdc84ce72202478ba1cec5cb557b71163de
PS C:\TEST> $pseditor.EditorServicesVersion


Major  Minor  Build  Revision
-----  -----  -----  --------
1      0      0      0


PS C:\> code --list-extensions --show-versions

be5invis.vscode-icontheme-nomo-dark@1.3.0
christian-kohler.path-intellisense@1.4.2
eamodio.gitlens@3.4.9
eridem.vscode-postman@0.5.1
Ionide.Ionide-fsharp@2.25.12
ms-vscode.csharp@1.9.0
ms-vscode.PowerShell@1.0.0
Shan.code-settings-sync@2.7.0
Tyriar.theme-sapphire@0.2.1
PS C:\TEST> $PSVersionTable


Name                           Value
----                           -----
PSVersion                      5.1.14393.1066
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.1066
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1


PS C:\>
```

### Issue Description
Not sure if this is a PowerShell Extension problem, or if it is a posh-git problem (feel free to reject this).

Recently after updating ps extension and vs code (sorry, don't recall from/to what versions) I noticed that when I hit enter at the Integrated Console prompt while posh-git is loaded, the prompt is reduced to ">", until an object is output in the console.

### Demo
![2017-05-18_16-52-52](https://cloud.githubusercontent.com/assets/2597673/26209140/4d4e19da-3bec-11e7-9edc-1c2bfedc8591.gif)

### Attached Logs
[logs.zip](https://github.com/PowerShell/vscode-powershell/files/1011308/logs.zip)

**Comment by pcgeek86** [comment]
@csandfeld Have you tried to reproduce this issue in the PowerShell console host? I don't immediately have access to a Windows 10 instance, but might have time to repro this later. Thanks for all the details, including the GIF.

**Comment by csandfeld** [comment]
@pcgeek86 Problem does not exist in the regular (non-integrated) PowerShell console

**Comment by csandfeld** [comment]
Problem appear to been solved along with the other integrated console fixes in [v1.2.1](https://github.com/PowerShell/vscode-powershell/releases/tag/v1.2.1).
Thanks :-)

**Comment by daviwil** [comment]
Hey Christian, very glad to hear that, thanks for letting us know!
